statsmodels/multivariate/factor.py

```diff
@@ -0,0 +1,272 @@
+# -*- coding: utf-8 -*-
+
+from statsmodels.base.model import Model
+from .factor_rotation import rotate_factors, promax
+
+try:
+    import matplotlib.pyplot as plt
+    missing_matplotlib = False
+except ImportError:
+    missing_matplotlib = True
+
+if not missing_matplotlib:
+    from .plots import plot_scree, plot_loadings
+
+import numpy as np
+from numpy.linalg import eigh, inv, norm, matrix_rank
+import pandas as pd
+
+from statsmodels.iolib import summary2
+
+
+class Factor(Model):
+    """
+    Factor analysis
+
+    .. [1] Hofacker, C. (2004). Exploratory Factor Analysis, Mathematical Marketing.
+    http://www.openaccesstexts.org/pdf/Quant_Chapter_11_efa.pdf
+
+    Supported rotations:
+        'varimax', 'quartimax', 'biquartimax', 'equamax', 'oblimin',
+        'parsimax', 'parsimony', 'biquartimin', 'promax'
+
+    Parameters
+    ----------
+    endog : array-like
+        Variables in columns, observations in rows
+
+    n_factor : int
+        The number of factors to extract
+
+    method : str
+        Specify the method to extract factors.
+        'pa' - Principal axis factor analysis
+
+    """
+    def __init__(self, endog, n_factor, exog=None, method='pa', **kwargs):
```

Comment:
exog is not used, so we should remove it here and use exog=None in super call

Reply:
`**kwargs` are not forwarded to super
remove it so that it doesn't silently swallow wrong arguments.
It should be possible to add `missing` as specific keyword